charts/app-config/values-staging.yaml

```diff
@@ -1308,6 +1308,7 @@ govukApplications:
             secretKeyRef:
               name: signon-token-government-frontend-publishing-api
               key: bearer_token
+  - name: govspeak-preview
```

Comment:
**polish:** spacing

A line break before this entry would be good

Reply:
Thanks, will add this.